conmech/dataclass/obstacle_properties.py

```diff
@@ -2,6 +2,6 @@
 
```

Comment:
please rename file to `obstacle_properties.py`

Reply:
both files are renamed, is there a typo?